app/assets/javascripts/my_modules/repositories.js

```diff
@@ -143,6 +158,41 @@ var MyModuleRepositories = (function() {
     });
   }
 
+  function setSelectedItem() {
+    let versionsSidebar = FULL_VIEW_MODAL.find('.repository-versions-sidebar');
+    let currentId = FULL_VIEW_MODAL.find('.table').data('id');
+    versionsSidebar.find('.list-group-item').removeClass('active');
+    versionsSidebar.find(`[data-id="${currentId}"]`).addClass('active');
+  }
+
+  function createDestroySnapshot(actionPath, requestType) {
```

Comment:
maybe 'attachDestroySnapshot' would be better name?

Reply:
or 'initDestroySnapshot'

Reply:
This is dual action, for Create and Destroy